### PR TITLE
feat(#110): Add SonarCloud config file

### PR DESCRIPTION
**Pull request by andrablaj**
# Description

Adds a file containing the repo-level SonarCloud configuration for cht-interoperability.

See [the related documentation](https://docs.communityhealthtoolkit.org/contribute/code/static-analysis).

Closes [medic/interoperability#[110]](https://github.com/medic/cht-interoperability/issues/110)

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by andrablaj** [comment]
@garethbowen I will need your help to enable SonarCloud for this repo by following the steps in [this documentation](https://docs.communityhealthtoolkit.org/contribute/code/static-analysis/#adding-a-new-repo-to-sonarcloud).

File: .sonarcloud.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sonar.sources=.
+# Can have multiple comma-separated entries
+sonar.exclusions=**/test*/**/*
+
+sonar.tests=.
+# Can have multiple comma-separated entries
+sonar.test.inclusions=**/test*/**/*
```
Comment (njuguna-n) [inline .sonarcloud.properties:7]: Nit: add a new line at the end of the file

**Review by njuguna-n — APPROVED** [review]
Looks good. I have added one minor comment but it's not a blocker.

**Comment by garethbowen** [comment]
@andrablaj Sonar is now set up for this repo.

**Pull request merged**